gradient/cli/jobs.py

```diff
@@ -169,7 +169,14 @@ def common_jobs_create_options(f):
             "--jobEnv",
             "job_env",
             type=json_string,
-            help="Environmental variables ",
+            help="Environmental variables (deprecated)",
+            cls=common.GradientOption,
+        ),
```

Comment:
I'd probably just remove this. If you want to support the old flag you can just add it to the `--envVars` option.